### PR TITLE
Adding Flag for Diffuse Texture Alpha

### PR DESCRIPTION
**Pull request by Pryme8**
Hopefully adding support to obj import mtl parsing to support a flag for diffuse texture having an alpha pass.
"bjs_diffuseHasAlpha 1"

### CODE REVIEW

File: loaders/src/OBJ/babylon.objFileLoader.ts [hunk 1]
```diff
@@ -121,7 +121,14 @@ module BABYLON {
                 } else if (key === "map_d" && material) {
                     // The dissolve of the material
                     material.opacityTexture = MTLFileLoader._getTexture(rootUrl, value, scene);
-
+                } else if (key == 'bjs_diffuseHasAlpha' && ((material) && material.diffuseTexture)){
```
Comment (deltakosh) [inline loaders/src/OBJ/babylon.objFileLoader.ts]: So you want to go with a custom parameter? This does not seem scalable

Comment (Pryme8) [reply loaders/src/OBJ/babylon.objFileLoader.ts]: Let me review the mtl file structure and and see if there is a parameter that covers the diffuse having an alpha.  I'm pretty sure there is not, so my thought would be you could edit your mtl after export.

What thoughts do you have?

Comment (deltakosh) [reply loaders/src/OBJ/babylon.objFileLoader.ts]: I get your point but this is kind of not convenient
I would perhaps prefer loading the texture and trying to check if there are alpha data

But your solution is also acceptable

Comment (Pryme8) [reply loaders/src/OBJ/babylon.objFileLoader.ts]: roger that! Let me whip up a method for that, then maybe we can compare and then get input from others?

**Comment by deltakosh** [comment]
Excellent idea!

**Comment by RaananW** [comment]
Hi @Pryme8 ,
Care to update what's new with the change? You can add it to this PR. Please don'T forget to update the documentation site (if not already done).
Thanks!!!

**Comment by Pryme8** [comment]
I have no way of testing this and have just kinda given up on the idea... so y'all can delete this PR. *sigh* sorry.

**Pull request closed without merging**